### PR TITLE
RUM - Next.js Pattern

### PR DESCRIPTION
**Pull request by lukebarrett24**

### CODE REVIEW

File: rum/getting-started.mdx [hunk 1]
```diff
@@ -151,6 +153,105 @@ To access your Session Replay data, navigate to the APM section, select the Trac
...
+        import Script from 'next/script'
+        
+        export default function MyApp({ Component, pageProps }) {
+        return (
+            <>
+            <Component {...pageProps} />
+            <Script
+                strategy="beforeInteractive"
+                id="middleware-lib-init"
+                src="https://cdnjs.middleware.io/browser/libs/0.0.1/stage/middleware-rum.min.js"
```
Comment (Archish27) [inline rum/getting-started.mdx:227]:
```suggestion
                src="https://cdnjs.middleware.io/browser/libs/0.0.1/middleware-rum.min.js"
```

File: rum/getting-started.mdx [hunk 2]
```diff
@@ -151,6 +153,105 @@ To access your Session Replay data, navigate to the APM section, select the Trac
...
+        children: React.ReactNode
+        }) {
+        
+        return (
+            <html lang="en">
+            <head>
+                <Script
+                strategy="beforeInteractive"
+                id="middleware-lib-init"
+                src="https://cdnjs.middleware.io/browser/libs/0.0.1/stage/middleware-rum.min.js"
```
Comment (Archish27) [inline rum/getting-started.mdx:181]:
```suggestion
                src="https://cdnjs.middleware.io/browser/libs/0.0.1/middleware-rum.min.js"
```

File: rum/getting-started.mdx [hunk 3]
```diff
@@ -151,6 +153,105 @@ To access your Session Replay data, navigate to the APM section, select the Trac
...
+                type="text/javascript"
+                />
+                <Script
+                strategy="beforeInteractive"
+                id="middleware-script-init">
+                {
+                    `
+                    if (window.Middleware){
+                        window.Middleware.track({
+                        serviceName: "name",
```
Comment (Archish27) [inline rum/getting-started.mdx:237]: nextjs-service-name

File: rum/getting-started.mdx [hunk 4]
```diff
@@ -151,6 +153,105 @@ To access your Session Replay data, navigate to the APM section, select the Trac
...
+                        projectName: "name",
```
Comment (Archish27) [inline rum/getting-started.mdx:238]: nextjs-project-name

File: rum/getting-started.mdx [hunk 5]
```diff
@@ -151,6 +153,105 @@ To access your Session Replay data, navigate to the APM section, select the Trac
...
```
Comment (Archish27) [inline rum/getting-started.mdx:192]:
```suggestion
                        serviceName: "nextjs-service-name",
                        projectName: "nextjs-project-name",
```

**Review by Zamichel — APPROVED** [review]
LGTM

**Pull request merged**